src/utils/utils_concurrency.h

```diff
@@ -27,10 +29,17 @@
 #else /* __cplusplus */
 #include <atomic>
 #define _Atomic(X) std::atomic<X>
+
+using std::memory_order_acq_rel;
+using std::memory_order_acquire;
+using std::memory_order_relaxed;
+using std::memory_order_release;
+
```

Comment:
Why this is needed. This looks like this should not bellong to this file

Reply:
moved to test/base.hpp